svg/renderer.go

```diff
@@ -397,3 +365,57 @@ func (r *SVG) RenderImage(img image.Image, m canvas.Matrix) {
 	r.writeClasses(r.w)
 	fmt.Fprintf(r.w, `"/>`)
 }
+
+func writeImageBytes(img image.Image, mimetype string, w io.Writer) error {
```

Comment:
This function is called only once, can we put it in the function above directly?